core/src/util/usFrameworkPrivate.cpp

```diff
@@ -168,6 +169,8 @@ void FrameworkPrivate::Shutdown(bool restart)
   case Bundle::STATE_STOPPING:
     // Shutdown already inprogress
     break;
+  default:
```

Comment:
I would prefer to not use a `default` label and rely on the compiler errors if we don't handle a case explicitly.